frontend/src/features/public-form/components/FormEndPage/FormEndPageContainer.tsx

```diff
@@ -9,25 +9,45 @@ import { usePublicFormContext } from '~features/public-form/PublicFormContext'
 import { FeedbackFormInput } from './components/FeedbackBlock'
 import { FormEndPage } from './FormEndPage'
 
-export const FormEndPageContainer = (): JSX.Element | null => {
+export const FormEndPageContainer = ({
+  isPreview,
+}: {
+  isPreview?: boolean
+}): JSX.Element | null => {
   const { form, formId, submissionData } = usePublicFormContext()
   const { submitFormFeedbackMutation } = usePublicFormMutations(formId)
   const toast = useToast()
   const [isFeedbackSubmitted, setIsFeedbackSubmitted] = useState(false)
 
+  /**
+   * Handles feedback submission
+   * @param isPreview whether form is in preview mode
+   */
   const handleSubmitFeedback = useCallback(
     (inputs: FeedbackFormInput) => {
+      // no mutation required in preview-form mode
+      if (isPreview) {
+        toast({
+          description: 'Thank you for submitting your feedback!',
```

Comment:
maybe we can update the toast to be more descriptive that this is a preview. 

```suggestion
          description: 'Thank you for submitting your feedback! Since you are in preview mode, the feedback is not stored.',
```